crates/ursa-proxy/src/config.rs

```diff
@@ -22,21 +22,26 @@ pub struct ProxyConfig {
 #[derive(Deserialize, Serialize, Debug)]
 pub struct ServerConfig {
     pub proxy_pass: String,
-    pub listen_addr: Option<String>,
-    pub listen_port: Option<u16>,
+    pub listen_addr: String,
     pub cert_path: String,
     pub key_path: String,
 }
 
 #[derive(Deserialize, Serialize, Debug, Clone)]
 pub struct MokaConfig {
+    pub max_capacity: u64,
     pub stream_buf: u64,
+    pub time_to_idle: u64,
+    pub time_to_live: u64,
 }
 
 impl Default for MokaConfig {
     fn default() -> Self {
         Self {
-            stream_buf: 2_000_000, // 2MB
+            max_capacity: 200_000_000,   //  Number of entries.
+            stream_buf: 1_000_000_000,   // 1GB.
```

Comment:
Is `stream_buf` the proxies chunking size ?

Reply:
Chunking size? 

This is what it's for https://docs.rs/tokio/latest/tokio/io/fn.duplex.html
> The max_buf_size argument is the maximum amount of bytes that can be written to a side before the write returns Poll::Pending.

Reply:
I was thinking it was the chunk size used when streaming data to the client. No worries, seems like it's just an arbitrary number